src/database/DataTypes/Location.js

```diff
@@ -5,13 +5,105 @@
 
 import Realm from 'realm';
 
+import { extractBreaches } from '../../utilities/modules/vaccines';
+import { getTotal } from '../utilities';
+
 export class Location extends Realm.Object {
-  // TODO
-  // getBreaches(database) <- ('SensorLogs') filter this ID and isInBreach = true
-  // getBatches ?
+  // TODO remove data object
+  data = {
```

Comment:
can remove

Reply:
i've removed it in: https://github.com/openmsupply/mobile/compare/%23874-Vaccine-Module-Page...%23874-Refinements]-Vaccine-Modul-Page, still one getter requires data in this PR